cmd/snap-confine/tests/common.sh

```diff
@@ -60,7 +60,7 @@ SHM="$(mktemp -d -p /run/shm)"
 trap 'rm -rf $TMP $SHM' EXIT
 
 # name snap-confine as the test name for improved logging
-L="$TMP/`basename $0`"
+L="$TMP/$(basename "$0")"
```

Comment:
only do:
L=$TMP/$(basename "$0")

there shoudl be no need for the outer quotes that makes it more confusing to read...